apps-rendering/src/components/Callout/calloutContact.tsx

```diff
@@ -4,7 +4,7 @@ import type { FC } from 'react';
 import { calloutLinkContainer, calloutPrimaryButton, info } from './styles';
 
 const capitalizeFirstLetter = (s: string): string =>
-	s.charAt(0).toUpperCase() + s.slice(1);
+	s === 'whatsapp' ? 'WhatsApp' : s.charAt(0).toUpperCase() + s.slice(1);
```

Comment:
should we compare on `s.toLowerCase()` do you think? Not sure where this data comes from so maybe it'll always be lower case?